src/adhocracy_mercator/adhocracy_mercator/resources/mercator2.py

```diff
@@ -0,0 +1,155 @@
+"""Mercator 2 proposal."""
+
```

Comment:
FYI: In the frontend we moved all the old code to a 2015 namespace and added a 2016 namespace for all new code.